api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -6206,6 +6232,13 @@ message AccessListDelete {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
+
+  // AccessList is common access list metadata.
+  AccessListMetadata AccessList = 4 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
```

Comment:
I don't love it's after the `Status` field, but that way it's backward compatible. (this isn't the only thing I introduced here I don't love about the field ordering)